packages/config/src/options.js

```diff
@@ -281,6 +281,11 @@ export function getNuxtConfig(_options) {
     consola.warn('vendor has been deprecated due to webpack4 optimization')
   }
 
+  if (options.build.parallel && typeof options.build.extractCSS !== 'undefined') {
```

Comment:
```suggestion
  if (options.build && options.build.parallel && typeof options.build.extractCSS !== 'undefined') {
```

Maybe? ☺️

Reply:
Although it is rare to not have `build` option, but still a potential read hazard, kudos.